### DISCUSSION TITLE
카카오, 네이버 동시 실행

### DISCUSSION THREAD

**Discussion by hky0102** [kimytsc/covid-rest-vaccine-macro#66] [Q&A]
안녕하세요,
카카오와 네이버 백신 예약 매크로창을 동시에 띄어 놓아도 될까요?

**Comment by kimytsc** [comment]
카카오와 네이버를 둘 중 한 곳에서 먼저 예약이 잡히면 다른 쪽에서 예약 시도시 이미 예약되어 실패하기 때문에 동시에 띄워놓으셔도 무관할 것 같습니다.
